web/source/build.sh

```diff
@@ -312,21 +313,21 @@ generate_environment_ts_file ( ) {
 
     echo "//Autogenerated file - do not modify!
 namespace com.keyman.environment {
-  export var VERSION = \"$VERSION\";
+  export var VERSION = \"$VERSION_RELEASE\";
```

Comment:
It would probably be best to forward `$VERSION_PATCH` through this auto-generated file as well; otherwise, the value doesn't get set for unminified versions.  (It's set for minified regardless.)

Reply:
Toward that end, please check out #2559.  It's a small patch that resolves the issue easily.  Feel free to auto-merge it if you like.

Reply:
Don't worry about it for 13.0.